jdaviz/cli.py

```diff
@@ -48,11 +48,17 @@ def main(filename, layout='default', instrument=None, browser='default',
         import asyncio
         asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
 
-    # Support comma-separate file list
-    filepath = ','.join([str(pathlib.Path(f).absolute()).replace('\\', '/')
-                         for f in filename.split(',')])
+    if filepaths:
+        # Convert paths to posix string; windows paths are not JSON compliant
+        file_list = [pathlib.Path(f).absolute().as_posix() for f in filepaths]
+        if layout == "imviz":
+            # Imviz multiloading should be done all at once for batch processing.
+            # Imviz convention is a single string of consecutive, comma-separated file paths
+            file_list = [','.join(file_list)]
```

Comment:
If you really don't like this, you can also modify the Imviz parser itself, though this is probably easier. So 👍 from me.

Reply:
I think we should revisit this in a separate push to incorporate batch processing across jdaviz. I know @kecnry suggested just using `jdaviz.batch_load` now, but I'm cautious to do that now (For ex. what if someone provides multiple mosviz directories? Does the `instrument` keyword get passed properly?). I'd prefer to keep the PRs smaller and scope-focused